src/ralph/ssl_certificates/models.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+from dj.choices import Choices
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.assets.models import BaseObject
+from ralph.assets.models.assets import AssetHolder
+from ralph.lib.mixins.models import AdminAbsoluteUrlMixin
+
+
+class CertificateType(Choices):
+    _ = Choices.Choice
+
+    ev = _('EV')
+    ov = _('OV')
+    dv = _('DV')
+    wildcard = _('Wildcard')
+    multisan = _('Multisan')
+    internal = _('CA ENT')
+
+
+class SSLCertificate(AdminAbsoluteUrlMixin, BaseObject):
+    name = models.CharField(
+        verbose_name=_('certificate name'),
+        help_text=_('Full certificate name'),
+        max_length=255
+    )
+    certificate_type = models.PositiveIntegerField(
+        choices=CertificateType(),
+        default=CertificateType.ov.id,
+    )
+    business_owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='certificates_business_owner',
+        blank=True,
+        null=True,
+        help_text=_('Business contact person for a certificate')
+    )
+    technical_owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='certificates_technical_owner',
+        blank=True,
+        null=True,
+        help_text=_('Technical contact person for a certificate')
+    )
+    issued_by = models.ForeignKey(
+        AssetHolder,
```

Comment:
are you sure that `AssetHolder` is the right model here? We're using it for `property_of` like fields (for storying companies like Allegro and similar), ex. https://github.com/allegro/ralph/blob/685fbfc9adec31d577aa21f32f1d7bb1572521e0/src/ralph/supports/models.py#L112-L117

Reply:
I see it. If I change it for:
issued_by= models.ForeignKey(
        Manufacturer,
        on_delete=models.PROTECT,
        blank=True,
        null=True,
    )

Then will be ok?

Reply:
yes